compile-wat.js

```diff
@@ -0,0 +1,29 @@
+import { resolve } from "path";
+import { readFileSync } from "fs";
+const initWabt = require("wabt");
```

Comment:
You may want to change this to `import * as initWabt from 'wabt'` or `import initWabt from 'wabt'` (whichever works)